Index: src/main/java/ITSOL/Bus/Management/Full/Stack/DAO/Entity/Route.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ITSOL.Bus.Management.Full.Stack.DAO.Entity;\r\n\r\nimport lombok.AllArgsConstructor;\r\nimport lombok.Data;\r\nimport lombok.NoArgsConstructor;\r\n\r\n@Data\r\n@AllArgsConstructor\r\n@NoArgsConstructor\r\npublic class Route {\r\n    private int route_ID;\r\n    private String distance;\r\n    private String stations;\r\n    private int isdeleted;\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/ITSOL/Bus/Management/Full/Stack/DAO/Entity/Route.java b/src/main/java/ITSOL/Bus/Management/Full/Stack/DAO/Entity/Route.java
--- a/src/main/java/ITSOL/Bus/Management/Full/Stack/DAO/Entity/Route.java	(revision 9c5c37be957f57cf53e14cf49e7286e761cb88ab)
+++ b/src/main/java/ITSOL/Bus/Management/Full/Stack/DAO/Entity/Route.java	(date 1647505506578)
@@ -9,7 +9,7 @@
 @NoArgsConstructor
 public class Route {
     private int route_ID;
-    private String distance;
-    private String stations;
+    private float distance;
+    private int stations;
     private int isdeleted;
 }
